src/components/Contacts/ContactDetails/ContactDetailsTab/Mailing/ContactDetailsTabMailing.tsx

```diff
@@ -0,0 +1,98 @@
+import { Box, Link, styled, Typography } from '@material-ui/core';
+import { LocationOn } from '@material-ui/icons';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { ContactDetailsTabQuery } from '../ContactDetailsTab.generated';
+
+const ContactDetailsMailingTitle = styled(Typography)(({ theme }) => ({
+  marginTop: theme.spacing(2),
+}));
+
+const ContactDetailsMailingMainContainer = styled(Box)(({ theme }) => ({
+  display: 'flex',
+  flexDirection: 'row',
+  marginTop: theme.spacing(2),
+}));
+
+const ContactDetailsMailingIcon = styled(LocationOn)(({ theme }) => ({
+  margin: theme.spacing(1),
+  width: '14px',
+  height: '20px',
+  flexGrow: 1,
+}));
+
+const ContactDetailsMailingTexContainer = styled(Box)(({}) => ({
+  flexGrow: 8,
```

Comment:
This feels like a really large number for a flexGrow.. why was 8 the magic number? Also I think you can add the flex properties directly as a prop on the ```Box``` components.

```jsx
<ContactDetailsMailingTextContainer flex flexGrow="8" />
```

BUT! At the same time, that could get very messy especially with these large component trees. So I think having these styled-components would be the better option, but let me know what you think.

Reply:
@TheNoodleMoose it didn't make a big deal on web but when I testing on a mobile screen 8 was the magic number.